docs/user/usage.md

```diff
@@ -12,12 +12,12 @@ After having [installed the requirements and built](../installation/build.md) sp
 
 The docker image can also be used directly, as detailed in the specific [docker documentation page](docker.md).
 
-The Nano S is the default model; the Nano X and Blue can be specified on the
-command-line:
+With applications built by recent SDKs, Speculos can automatically detect the targeted device. Otherwise, the Nano S is the default; the Nano X, Stax and Blue can be specified on the command line:
```

Comment:
```suggestion
With applications built by recent SDKs, Speculos can automatically detect the targeted device. Otherwise, the Nano S is the default; the Nano X, Nano S+, Stax and Blue can be specified on the command line:
```

Reply:
Thanks, I also added the Nano S+ to the SDK matrix compatibility.

Reply:
Ok thanks, technically the matrix is not up to date:
- it lacks recent OS/SDK which are supported
- for recent OS/SDK, the parameter is now -a/--apiLevel and not -k/--sdk anymore.

but all of this don't matter anymore as Speculos detect it autonomously too on apps build with recent SDK.